colcon_core/python_install_path.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+# Licensed under the Apache License, Version 2.0
+
+from pathlib import Path
+import sysconfig
+
+
+def get_python_install_path(name, vars_=None):
+    """
+    Get Python install paths matching Colcon's preferred scheme.
+
+    See sysconfig.get_path for more info about the arguments.
+
+    :param name: Name of the path type
+    :param vars_: A dictionary of variables updating the values of
+        sysconfig.get_config_vars()
+    :rtype: Pathlib.Path
+    """
+    kwargs = {}
+    if vars_:
+        kwargs['vars'] = vars_
```

Comment:
Do you mind if we drop this for the sake of test coverage? The case is handled in CPython:
https://github.com/python/cpython/blob/1748816e80b23744667e239b49b477c0e283d201/Lib/sysconfig.py#L214-L218
```suggestion
    kwargs = {'vars': vars_}
```

Reply:
In 6329f54 I took a avoided the conditional with a tuple to avoid assuming `sysconfig` will accept `vars` being `None`, since the documentation says `If vars is provided, it must be a dictionary`.